install/load_default_modules.sh

```diff
@@ -2,25 +2,19 @@
 module load slurm
 
 if [[ `hostname` =~ 'mahuika' ]] || [[ $HOSTNAME =~ 'wb' ]];then
-    module load Python/2.7.14-gimkl-2017a
+    load_python3_mahuika
     module load PrgEnv-cray/1.0.4
     module load GDAL/2.2.2-gimkl-2017a-GEOS-3.5.1
     module del LibTIFF/4.0.7-gimkl-2017a
     #python libs
-    export PYTHONPATH=/nesi/project/nesi00213/opt/mahuika/python-packages/lib/python2.7/site-packages:$PYTHONPATH
     export PATH=/nesi/project/nesi00213/opt/mahuika/python-packages/bin:$PATH
-    #qcore
-    export PYTHONPATH=/nesi/project/nesi00213/opt/mahuika/qcore:$PYTHONPATH
     #gmt
     export PATH=/nesi/project/nesi00213/opt/mahuika/gmt/5.4.4/bin:$PATH
     export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/nesi/project/nesi00213/opt/mahuika/gmt/5.4.4/lib64
 elif [[ `hostname` =~ 'maui' ]] || [[ $HOSTNAME =~ 'ni' ]];then 
-    module load cray-python/2.7.15.1
+    load_python3_maui
     #python libs
-    export PYTHONPATH=/nesi/project/nesi00213/opt/maui/python-packages/lib/python2.7/site-packages:$PYTHONPATH
     export PATH=/nesi/project/nesi00213/opt/maui/python-packages/bin:$PATH
-    #qcore
-    export PYTHONPATH=/nesi/project/nesi00213/opt/maui/qcore:$PYTHONPATH
```

Comment:
I'm pretty sure the qcore libs are not 'installed' in the virtual environment, yet.
so to remove this line, we either have to setup the libs properly;
or we kept this line, but modify it so both maui/mahuika uses the same path (since the load_config in qcore now determines the machine, and we no longer need two different qcore libs)

Reply:
The load_python3 line sets up the python path correctly for the virtual environment.

Reply:
But qcore is not installed in the virtual_env.